stable/envoy/templates/deployment.yaml

```diff
@@ -88,6 +88,8 @@ spec:
             - name: {{ $key }}
               mountPath: {{ $value.mountPath }}
             {{- end }}
+          lifecycle:
+            {{ toYaml .Values.lifecyle | nindent 12 }}
```

Comment:
This doesn't work as `lifecyle` != `lifecycle`. Thanks!

Reply:
Thanks for report!

I fix this typo at #23258
Please use version after 1.9.1